qutebrowser/browser/commands.py

```diff
@@ -1214,23 +1214,43 @@ def bookmark_del(self, url=None):
 
     @cmdutils.register(instance='command-dispatcher', name='inspector',
                        scope='window')
-    def toggle_inspector(self):
+    @cmdutils.argument('position', choices=['right', 'left', 'top', 'bottom',
+                                            'window'])
+    def toggle_inspector(self, position=None):
         """Toggle the web inspector.
 
         Note: Due a bug in Qt, the inspector will show incorrect request
         headers in the network tab.
+
+        Args:
+            position: Where to open the inspector
+                      (right/left/top/bottom/window).
         """
         tab = self._current_widget()
         # FIXME:qtwebengine have a proper API for this
         page = tab._widget.page()  # pylint: disable=protected-access
 
+        @pyqtSlot()
+        def _on_inspector_closed():
+            tab.data.inspector = None
```

Comment:
What's this used for? Originally the inspect would be reused and not created anew when it's closed and reopened.

Reply:
I probably didn't think about reusing the inspector and the code needed to handle the case where no inspector is initialized anyway so it was simpler to implement.

I'd surely appreciate a solution which reuses the inspector on reopen.